Symfony/CS/Tokenizer/Token.php

```diff
@@ -50,6 +50,26 @@ class Token
     private $changed = false;
 
     /**
+     * Get classy tokens: T_CLASS, T_INTERFACE or T_TRAIT (if defined).
```

Comment:
As the method returns an array with all classy tokens, I would prefer "and" instead of "or".